static/app/components/events/searchBar.tsx

```diff
@@ -203,7 +219,18 @@ function SearchBar(props: SearchBarProps) {
       kind: FieldKind.FIELD,
     };
 
-    return omit(combinedTags, omitTags ?? []);
+    const list = omit(combinedTags, omitTags ?? []);
+
+    if (transaction) {
+      const totalCount: number = Object.keys(list).length;
+      transaction.setTag('tags.totalCount', totalCount);
+      const countGroup = [
+        1, 5, 10, 20, 50, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000,
```

Comment:
crank the last one to like `100000` in case I was wrong about max tags roughly and it's the cause.